doc/input-output-reference/src/overview/group-setpoint-managers.tex

```diff
@@ -195,7 +195,18 @@ \subsubsection{Inputs}\label{inputs-2-037}
 
 \paragraph{Field: Control Variable}\label{field-control-variable-2}
 
-The type of variable that will be controlled. There is only one choice for this type of setpoint manager: Temperature.
+The type of variable that will be controlled. There are three key choices for this type of set point manager:
+
+\begin{itemize}
+\item
+  Temperature
+\item
+  MaximumTemperature
+\item
+  MinimumTemperature
+\end{itemize}
+
+If the Control Variable input field is set to MaximumTemperature, then the setpoint will be set to a setpoint value specified at Setpoint at Outdoor Low Temperature input field, else if the Control Variable input field is set to MinimumTemperature, then the setpoint will be set to a setpoint value specified in Setpoint at Outdoor High Temperature input field, else if the Control Variable input field is set to Temperature, then the setpoint will be set to a setpoint value determined based on the current time outdoor air temperature, and the setpoint values specified at Setpoint Outdoor Low and High Temperature input fields.
```

Comment:
Also add a note at the end of the DualSetpoint paragraph:  Note that DualSetpoint is used only for plant loops with both heating and cooling supply equipment, such as a water loop heat pump system. When the loop temperature rises above the MaximumTemperature setpoint, then the loop will request cooling, and when the loop temperature is below the MinimumTemperature it will request heating.  DualSetpoint cannot be used to control a single-mode loop, such as a hot water heating loop between operating between high and low boiler setpoints.